migration-roadmap/infield-button.md

```diff
@@ -0,0 +1,414 @@
+        <sp-icon-add class="spectrum-InfieldButton-icon"></sp-icon-add>
+    </div>
+</button>
+```
+
+**Legacy (CSS main branch) - Stacked buttons:**
+
+```html
+<button
+    class="spectrum-InfieldButton spectrum-InfieldButton--sizeM spectrum-InfieldButton--top"
+    aria-haspopup="listbox"
+    type="button"
+    tabindex="0"
+    aria-label="add"
+>
+    <div class="spectrum-InfieldButton-fill">
+        <sp-icon-chevronup75
+            class="spectrum-InfieldButton-icon"
+        ></sp-icon-chevronup75>
+    </div>
+</button>
+<button
+    class="spectrum-InfieldButton spectrum-InfieldButton--sizeM spectrum-InfieldButton--bottom"
+    aria-haspopup="listbox"
+    type="button"
+    tabindex="0"
+    aria-label="add"
+>
+    <div class="spectrum-InfieldButton-fill">
+        <sp-icon-chevrondown75
+            class="spectrum-InfieldButton-icon"
+        ></sp-icon-chevrondown75>
+    </div>
+</button>
+```
+
+**Spectrum 2 (CSS spectrum-two branch) - Single button:**
+
+```html
+<button
+    class="spectrum-InfieldButton spectrum-InfieldButton--sizeM"
+    aria-haspopup="listbox"
+    type="button"
+    tabindex="0"
+    role="presentation"
+>
+    <div class="spectrum-InfieldButton-fill">
+        <sp-icon-add100 class="spectrum-InfieldButton-icon"></sp-icon-add100>
+    </div>
+</button>
+```
+
+**Spectrum 2 (CSS spectrum-two branch) - Inline buttons:**
+
+```html
+<div class="spectrum-InfieldButton-inline">
+    <button
+        class="spectrum-InfieldButton spectrum-InfieldButton--sizeM"
+        aria-haspopup="listbox"
+        type="button"
+        tabindex="0"
+        aria-label="minus"
+    >
+        <div class="spectrum-InfieldButton-fill">
+            <sp-icon-dash class="spectrum-InfieldButton-icon"></sp-icon-dash>
+        </div>
+    </button>
+    <button
+        class="spectrum-InfieldButton spectrum-InfieldButton--sizeM"
+        aria-haspopup="listbox"
+        type="button"
+        tabindex="0"
+        aria-label="add"
+    >
+        <div class="spectrum-InfieldButton-fill">
+            <sp-icon-add class="spectrum-InfieldButton-icon"></sp-icon-add>
+        </div>
+    </button>
+</div>
+```
+
+### Key Changes in HTML Structure
+
+1. **Layout paradigm shift**: Replaced `isStacked` vertical stacking with `isInline` horizontal grouping
+2. **Enhanced iconography**:
+    - Single button: Uses size-specific icon names (e.g., `Add100` instead of `Add`)
+    - Inline buttons: Switched from chevron icons to more intuitive action icons (`Dash`/`Add`)
+3. **Simplified state management**: Removed `isInvalid` and `isFocused` states while preserving `isHovered`/`isActive`
+4. **Improved accessibility**: Added `role="presentation"` for single buttons and proper `aria-label` differentiation for inline buttons
+5. **Container structure**: Inline variant wraps buttons in `spectrum-InfieldButton-inline` container
+
+</details>
+### CSS => SWC mapping
+@@ -15,7 +12,6 @@ export const Template = (
```

Comment:
Hallucination! Could you remove this first CSS => SWC mapping section? I think it's just the template.js file.